packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -23,105 +22,163 @@ import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
+    // mapping(GameType => OutputRoot) public anchors;
+
+    // TODO: determine if this belongs
+    // uint256 __gap0;
 
     /// @notice Address of the SuperchainConfig contract.
     ISuperchainConfig public superchainConfig;
 
-    /// @param _disputeGameFactory DisputeGameFactory address.
-    constructor(IDisputeGameFactory _disputeGameFactory) {
-        DISPUTE_GAME_FACTORY = _disputeGameFactory;
+    /// @notice Returns whether a game is blacklisted.
+    mapping(IDisputeGame => bool) public isGameBlacklisted;
+
+    uint64 public gameRetirementTimestamp;
+
+    GameType public respectedGameType;
+
+    IFaultDisputeGame internal _anchorGame;
+
+    constructor() {
         _disableInitializers();
     }
 
     /// @notice Initializes the contract.
-    /// @param _startingAnchorRoots An array of starting anchor roots.
+    /// @param _disputeGameFactory DisputeGameFactory address.
+    /// @param _disputeGameFinalityDelaySeconds The delay between when a dispute game is resolved and when it can be
+    /// considered finalized.
+    /// @param _authorizedGame An authorized dispute game.
     /// @param _superchainConfig The address of the SuperchainConfig contract.
     function initialize(
-        StartingAnchorRoot[] memory _startingAnchorRoots,
+        IDisputeGameFactory _disputeGameFactory,
+        uint256 _disputeGameFinalityDelaySeconds,
+        IFaultDisputeGame _authorizedGame,
         ISuperchainConfig _superchainConfig
     )
         external
         initializer
     {
-        for (uint256 i = 0; i < _startingAnchorRoots.length; i++) {
-            StartingAnchorRoot memory startingAnchorRoot = _startingAnchorRoots[i];
-            anchors[startingAnchorRoot.gameType] = startingAnchorRoot.outputRoot;
-        }
+        disputeGameFactory = _disputeGameFactory;
+        disputeGameFinalityDelaySeconds = _disputeGameFinalityDelaySeconds;
+        _anchorGame = _authorizedGame;
         superchainConfig = _superchainConfig;
     }
 
-    /// @notice Returns the DisputeGameFactory address.
-    /// @return DisputeGameFactory address.
-    function disputeGameFactory() external view returns (IDisputeGameFactory) {
-        return DISPUTE_GAME_FACTORY;
+    /// @notice Returns the output root of the anchor game, or an authorized anchor state if no such game exists.
+    function anchors(GameType /* unused */ ) public view returns (OutputRoot memory) {
+        if (isGameBlacklisted[_anchorGame]) revert InvalidGame();
+        return
+            OutputRoot({ l2BlockNumber: _anchorGame.l2BlockNumber(), root: Hash.wrap(_anchorGame.rootClaim().raw()) });
     }
 
-    /// @notice Callable by FaultDisputeGame contracts to update the anchor state. Pulls the anchor state directly from
-    ///         the FaultDisputeGame contract and stores it in the registry if the new anchor state is valid and the
-    ///         state is newer than the current anchor state.
-    function tryUpdateAnchorState() external {
```

Comment:
we no longer need this method, is it ok to remove? old dispute games will still be using the old ASR.